src/main/java/org/jitsi/jicofo/rest/Health.java

```diff
@@ -138,6 +145,30 @@ static void getJSON(
         try
         {
             check(focusManager);
+
+            // At this point the health check has passed - now eventually list
+            // JVBs known to this Jicofo instance
+            String listJvbParam = request.getParameter(LIST_JVB_PARAM_NAME);
+
+            if (!StringUtils.isNullOrEmpty(listJvbParam)
+                && "true".equals(listJvbParam.toLowerCase()))
+            {
```

Comment:
Just a typo, maybe using Boolean.parseBoolean(listJvbParam) makes it more readable.